httpi.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'mocha',    '~> 0.13'
   s.add_development_dependency 'puma',     '~> 2.3.2'
   s.add_development_dependency 'webmock'
+  s.add_development_dependency 'multipart-post'
```

Comment:
Hi @keeguon! Sorry! The "http gems" should be on `Gemfile`, i didn't know too! 😳